Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,80 @@
+description: Unshorten urls through a tor proxy server.
+detaileddescription: Cloaken will return the unshortened version of a url;  Cloaken is available for install on AWS marketplace. Visit https://aws.amazon.com/marketplace/ search for Cloaken.  Follow the deployment instructions. 
+configuration:
+- display: ""
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: ""
+  name: server_url
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |
+    # The command demisto.command() holds the command sent from the user.
+    from cloakensdk.client import SyncClient
+    from cloakensdk.resources import Url
+
+    def get_client():
+        server = demisto.args(["server"])
+        username = demisto.args()["credentials"]["password"]
+        password  = demisto.args()["credentials"]["identifier"]
+        client = SyncClient(server_url=server,
+                                 username=username,
+                                 password=password)
+        return  client
+
+
+    if demisto.command() == 'test-module':
+        # This is the call made when pressing the integration test button.
+        client = get_client()
+        demisto.results('ok')
+        sys.exit(0)
+
+    if demisto.command() == "cloaken":
+        client = getclient()
+        url = demisto.args()["url"]
+        resource = Url(client)
+        response = resource.unshorten(url)
+        ec={}
+        ec["URL"]={}
+        ec["Cloaken"]=[]
+        context={}
+        if response["status"]=="Success":
+            url_data = response["data"]["unshortened_url"]
+            ec["URL"]["Data"]=url_data
+            context["original_url"]=url
+            context["unshortened_url"]
+            ec["Cloaken"].append(context)
+            demisto.results([{
```

Comment:
Please use `return_outputs()`

Reply:
I do not see return_outputs in the script helper.  What is the signature of this method?

Reply:
![image](https://user-images.githubusercontent.com/31018228/58453909-7b111180-8125-11e9-9810-3da8925963b2.png)

Reply:
done